04.ls/ls.rb

```diff
@@ -92,14 +88,10 @@ def get_group_name(stat)
   Etc.getpwuid(stat.gid).name
 end
 
-def get_size_string(stat)
-  stat.size.to_s
-end
-
 def display_file_names(file_names)
-  files_number = file_names.size.to_f
-  row_size = (files_number / COL_MAX).ceil
-  col_size = (files_number / row_size).ceil
+  files_count = file_names.size.to_f
```

Comment:
```suggestion
  file_count = file_names.size.to_f
```

この場合、fileはcountを形容する使い方になるので "s" は不要ですね。（英語の話）